scripts/test/SANS/gui_logic/main_presenter_test.py

```diff
@@ -21,7 +21,7 @@ def test_that_the_white_list_is_correct(self):
         presenter = MainPresenter(SANSFacility.ISIS)
         self.assertTrue(presenter.get_number_of_white_list_items() == 0)
         white_list = presenter.get_white_list(show_periods=True)
-        self.assertTrue(presenter.get_number_of_white_list_items() == 19)
+        self.assertEqual(presenter.get_number_of_white_list_items(), 20)
```

Comment:
is the increase to 20 just because you have added a row?

Reply:
This is because I added a new column yes.